plugins/module_utils/errors.py

```diff
@@ -0,0 +1,210 @@
+# -*- coding: utf-8 -*-
+
+# (c) 2023, Jan Kodera (jkodera@purestorage.com)
+# GNU General Public License v3.0+ (see COPYING.GPLv3 or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+try:
+    import fusion as purefusion
+except ImportError:
+    pass
+
+import sys
+import json
+import re
+
+
+class OperationException(Exception):
+    """Raised if an asynchronous Operation fails."""
+
+    def __init__(self, op):
+        self._op = op
+
+    @property
+    def op(self):
+        return self._op
+
+
+def _should_be_verbose(module) -> bool:
+    # verbosity is a private member and Ansible does not really allow
+    # providing extra information only if the user wants it due to ideological
+    # reasons, so extract it as carefully as possible and assume non-verbose
+    # if something fails
+    try:
+        if module._verbosity is not None and isinstance(module._verbosity, int):
```

Comment:
It's true that these two would semantically wise have (I think) the same outcome, but to tell the truth whenever I see `if x is not None:` it immediately says to me _that's presence check_ but when I see `if x:` that says to me _that is a value test_. This difference would confuse me here since the reason for this check here is because _verbosity is for unspecified reasons a private parameter and I wanted to be able to be semi-sure the code will not completely break if Ansible ever removes it. Because of this I would oppose changing it here.

Reply:
Okey dokey :-)